libraries/core_libs/storage/include/storage/storage.hpp

```diff
@@ -79,8 +73,10 @@ class DbStorage : public std::enable_shared_from_this<DbStorage> {
 
    public:
     size_t const ordinal_;
+    const rocksdb::Comparator* comparator_;
```

Comment:
I dont like this usage of raw pointer. I understand you probably need raw pointer for setting `rocksdb::ColumnFamilyOptions().comparator`, but it seems that you could use unique_ptr here ? :

- change raw ptr to unique_ptr
- make it private:
- create method that would return raw pointer from unique_ptr and use that for setting ColumnFamilyOptions
- getIntComparator() can return unique_ptr too

Reply:
uniqe_ptr does not make sense as more columns could use same compactor, I can create shared_ptr,  if you think is better

Reply:
well this comment was about usage of smart_pointers vs raw pointers... choose whatever type of smart pointer is suitable. We have two options:

- write this class according to the best practices(no raw pointers, not using methods we dont need like that kClassName(), ...)
- or we can use this copy paste style of comparator class from rocksdb that implemented and keep the style according to rocksdb

Reply:
Actually I do not see any point using shared_ptrs as this approached (copied from rocksdb) is kind of singleton and there is no need to use shared_ptrs over there

Reply:
let's discuss this over the meeting